src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -93,36 +161,99 @@ public void testGVCFModeIsConsistentWithPastResults() throws Exception {
+        // -ERC GVCF --out a.gatk3.5.noDownsample.g.vcf -G StandardHC -G Standard \
+        // --disableDithering --no_cmdline_in_header  -dt NONE --maxReadsInRegionPerSample 100000000 --minReadsPerAlignmentStart 100000
+        final File gatk3Output = new File(TEST_FILES_DIR + "expected.testGVCFMode.gatk3.5.g.vcf");
+
+        final String[] args = new String[] {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_reference_20_21,
+                "-L", "20:10000000-10100000",
+                "-O", output.getAbsolutePath(),
+                "-ERC", "GVCF",
+                "-pairHMM", "AVX_LOGLESS_CACHING"
+        };
+
+        runCommandLine(args);
+
+        final double concordance = calculateConcordance(output, gatk3Output);
+        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.5 in GVCF mode is < 99%");
+    }
+
+    @Test
+    public void testGVCFModeIsConcordantWithGATK3_5AlelleSpecificResults() throws Exception {
+        Utils.resetRandomGenerator();
+        final File output = createTempFile("testGVCFModeIsConcordantWithGATK3_5AlelleSpecificResults", ".g.vcf");
+        final File gatk3Output = new File(TEST_FILES_DIR + "expected.testGVCFMode.gatk3.5.alleleSpecific.g.vcf");
+
+        //Created by running:
+        // java -jar ~/bin/GenomeAnalysisTK-3.5.0/GenomeAnalysisTK.jar -T HaplotypeCaller \
+        // -I ./src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam \
+        // -R src/test/resources/large/human_g1k_v37.20.21.fasta -L 20:10000000-10100000 \
+        // -ERC GVCF --out as.gatk3.5.noDownsample.g.vcf -G StandardHC -G Standard -G AS_Standard \
+        // --disableDithering --no_cmdline_in_header  -dt NONE --maxReadsInRegionPerSample 100000000 --minReadsPerAlignmentStart 100000
 
         final String[] args = new String[] {
                 "-I", NA12878_20_21_WGS_bam,
                 "-R", b37_reference_20_21,
                 "-L", "20:10000000-10100000",
                 "-O", output.getAbsolutePath(),
+                "-G", "StandardAnnotation",
+                "-G", "AS_StandardAnnotation",
                 "-ERC", "GVCF",
                 "-pairHMM", "AVX_LOGLESS_CACHING"
         };
 
         runCommandLine(args);
 
         final double concordance = calculateConcordance(output, gatk3Output);
-        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.4 in GVCF mode is < 99%");
+        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.5 in AS GVCF mode is < 99%");
     }
 
     @Test
     public void testBamoutProducesReasonablySizedOutput() {
         Utils.resetRandomGenerator();
 
         // We will test that when running with -bamout over the testInterval, we produce
-        // a bam with a number of reads that is within 10% of what GATK3.4 produces with
+        // a bam with a number of reads that is within 10% of what GATK3.5 produces with
```

Comment:
Did you update `gatk3BamoutNumReads` in this test to reflect the number of reads that GATK3.5 produces with `-bamout`?